polaris-core/src/main/java/org/apache/polaris/core/persistence/PolarisMetaStoreManager.java

```diff
@@ -129,6 +130,43 @@ ListEntitiesResult listEntities(
       @Nonnull PolarisEntitySubType entitySubType,
       @Nonnull PageToken pageToken);
 
+  /**
+   * Load entities with pagination
```

Comment:
Let's expand this javadoc description just a bit to make it more obvious that this method is effectively a "list and load entities under a parent", and maybe include some statement pointing at `{@link #listEntities}` to make the intention clear of making this listing/pagination behavior here match that of `listEntities`.

Michael's mention of future additions of `loadEntitiesById` and `loadEntitiesByName` had just reminded me that the naming could feel ambiguous, so javadocs can save developers time.